### PR TITLE
fix #75669 mc-at.org

### LINKED ISSUE #75669: mc-at.org
### Issue URL (Anti Adblock Script)
 [https://mc-at.org](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmc-at.org)
### Comment
Username: `@Kled4`
> Mc-at sitesinde adguardın tespit edilmesi

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.23
Browser: | Opera
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Strip URLs from tracking parameters, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard German, <br/>Bulgarian list, <br/>Icelandic ABP List, <br/>Dandelion Sprout's Nordic Filters
Other extensions used: | adguard

### PR DESCRIPTION
**Pull request by wout-codes**
#75669

### CODE REVIEW

File: TurkishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -1,6 +1,9 @@
 !
 ! These rules are against all kinds of ad blocking reinjection/circumvention scripts
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75669
+mc-at.org#%#//scriptlet("prevent-setTimeout", "showads", "7000")
```
Comment (Alex-302) [inline TurkishFilter/sections/antiadblock.txt:5]: It can be closed. So this is annoyance.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply TurkishFilter/sections/antiadblock.txt]: @Alex-302 Fixed.

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -428,6 +428,8 @@ sonichits.com,thepostmillennial.com,muropaketti.com,visajourney.com,dennikn.sk,i
 @@||di.se/scripts/advertisement__*.js
 !#safari_cb_affinity
 !
+mc-at.org#%#//scriptlet("prevent-setTimeout", "showads", "7000")
+mc-at.org#@#.showads
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/antiadblock.txt:432]: This rule must be wrapped into` !#safari_cb_affinity(general)` directive

Please read about it here:
`https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#pre-processor-directives`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 2 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/antiadblock.txt:432]: @Sergey-Lyapin I actually don't understand the Python's `If directive` and the hints. Hints like `!+  !+ PLATFORM(ios, ext_android_cb, ext_safari)`. What's the difference and when should I use the correct one?

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/antiadblock.txt]: @wout-codes
"Platform" affects only one rule under the hint, but we can wrap several rules into if/endif - this is the main difference =)
Use "Platform" if it must affect only one rule

**Pull request merged**